sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2022_08_31/models/_form_recognizer_client_enums.py

```diff
@@ -33,12 +33,6 @@ class ContentType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     APPLICATION_OCTET_STREAM = "application/octet-stream"
     #: Content Type 'application/pdf'.
     APPLICATION_PDF = "application/pdf"
-    #: Content Type 'application/vnd.openxmlformats-officedocument.presentationml.presentation'.
-    APPLICATION_VND_OPENXMLFORMATS_OFFICEDOCUMENT_PRESENTATIONML_PRESENTATION = "application/vnd.openxmlformats-officedocument.presentationml.presentation"
-    #: Content Type 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'.
-    APPLICATION_VND_OPENXMLFORMATS_OFFICEDOCUMENT_SPREADSHEETML_SHEET = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
-    #: Content Type 'application/vnd.openxmlformats-officedocument.wordprocessingml.document'.
-    APPLICATION_VND_OPENXMLFORMATS_OFFICEDOCUMENT_WORDPROCESSINGML_DOCUMENT = "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
```

Comment:
Did we ever document support for these content-types that are being removed in this version?

Reply:
No I already checked 👍 but I'll make sure to do a full docs pass again soon.